NC-NEWS/src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+export const getArticles = () => {
+  return axios
+    .get("https://chazzat18-nc-news.onrender.com/api/articles")
+    .then((response) => {
+      return response.data.articles;
+    });
+};
```

Comment:
Great to see an extracted reusable util function in use, and using `Axios` will help you in the long run, especially when dealing with errors.